tests/test_cli.py

```diff
@@ -297,7 +297,7 @@ def test_command_add_data_sample_already_exists(workdir, mocker):
     ('traintuple', 'key', models.Traintuple),
     ('aggregatetuple', 'key', models.Aggregatetuple),
     ('composite_traintuple', 'key', models.CompositeTraintuple),
-    ('compute_plan', 'compute_plan_id', models.ComputePlan),
+    ('compute_plan', 'key', models.ComputePlan),
```

Comment:
The second arg value in these parametrized values is always "key" now, maybe you could remove it entirely?